src/testcafe-runner.ts

```diff
@@ -266,6 +286,15 @@ async function run(nodeBin: string, runCfgPath: string, suiteName: string) {
     suite
   } = await prepareConfiguration(nodeBin, runCfgPath, suiteName);
 
+  // TestCafe used a reverse proxy for browser automation before.
+  // With TestCafe 3.0.0 and later, native automation was adopted,
```

Comment:
"native automation" isn't really a thing afaik, but just a TestCafe marketing term. Would be good to leave a link to https://testcafe.io/documentation/404237/guides/intermediate-guides/native-automation-mode

Reply:
👍